android/build.gradle

```diff
@@ -13,8 +13,7 @@ buildscript {
         maven { url 'https://dl.bintray.com/android/android-tools/' }
     }
     dependencies {
-        classpath('com.android.tools.build:gradle:3.6.3')
-
+        classpath('com.android.tools.build:gradle:4.0.0')
```

Comment:
Unrelated change?

Reply:
I was unable to build the app to confirm that the name updates did not break the build without this update. Newer versions of the JDK require an updated gradle.